Makefile

```diff
@@ -55,6 +55,12 @@ OPERATOR_SDK_VERSION ?= v1.34.2
 
 # Image URL to use all building/pushing image targets
 IMG ?= quay.io/konveyor/oadp-operator:latest
+
+# TTL_DURATION defines the time-to-live for temporary images pushed to ttl.sh
+# The maximum allowed value by ttl.sh is 24h. Default is 1h.
+# You can override this with environment variable (e.g., export TTL_DURATION=4h)
+TTL_DURATION ?= 1h
```

Comment:
how about naming it:

And then check if the number is within 1-24?

```Makefile
TTL_DURATION_HOURS ?= 1

ifeq (,$(filter $(TTL_DURATION_HOURS),1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24))
$(error TTL_DURATION_HOURS must be between 1 and 24 (got: $(TTL_DURATION_HOURS)))
endif
```

And then when you use it simply add 'h' as the suffix. 

?

Reply:
well that is limiting. maybe I want 6h30s